buildSrc/src/main/java/GenerateQuestListTask.kt

```diff
@@ -37,9 +37,9 @@ open class GenerateQuestListTask : DefaultTask() {
     @TaskAction
     fun run() {
         val questFileContent = sourceDirectory.resolve("quests/QuestsModule.kt").readText()
-        val questNameRegex = Regex("(?<=^ {4})[A-Z][a-zA-Z]+(?=\\()", RegexOption.MULTILINE)
+        val questNameRegex = Regex("^\s+([A-Z][a-zA-Z]+)\\(", RegexOption.MULTILINE)
```

Comment:
This would also allow digits after the first alpha character which I believe is valid, even if we aren't currently using it.
```suggestion
        val questNameRegex = Regex("^\s+([A-Z][a-zA-Z0-9]+)\\(", RegexOption.MULTILINE)
```